x/ethbridge/types/msgs.go

```diff
@@ -17,24 +17,14 @@ const (
 	lockGasCost = 160000000000 * 393000
 )
 
-// MsgLock defines a message for locking coins and triggering a related event
-type MsgLock struct {
-	CosmosSender     sdk.AccAddress  `json:"cosmos_sender" yaml:"cosmos_sender"`
-	Amount           sdk.Int         `json:"amount" yaml:"amount"`
-	Symbol           string          `json:"symbol" yaml:"symbol"`
-	EthereumChainID  int             `json:"ethereum_chain_id" yaml:"ethereum_chain_id"`
-	EthereumReceiver EthereumAddress `json:"ethereum_receiver" yaml:"ethereum_receiver"`
-	CethAmount       sdk.Int         `json:"ceth_amount" yaml:"ceth_amount"`
-}
-
 // NewMsgLock is a constructor function for MsgLock
 func NewMsgLock(
-	ethereumChainID int, cosmosSender sdk.AccAddress,
+	ethereumChainID int64, cosmosSender sdk.AccAddress,
```

Comment:
Should this be a uint64 instead of an int64? I'm having a hard time imagining a chainid or chain descriptor being negative.